monasca-api-python/Dockerfile

```diff
@@ -43,7 +43,6 @@ RUN apk add --no-cache python py2-pip py2-jinja2 curl mysql-client && \
   apk del build-dep
 
 COPY api-* /etc/monasca/
-COPY template.py start.sh  /
```

Comment:
Ah, good catch. I should probably not be allowed to merge things by hand before I've had coffee :smiley: